mapboxgl/utils.py

```diff
@@ -9,7 +9,7 @@ def df_to_geojson(df, properties=None, lat='lat', lon='lon', precision=None):
         df[lon] = df[lon].round(precision)
 
     if not properties:
-        properties = list(df.columns)
+        properties = [c for c in df.columns if c not in [lat, lon]]
```

Comment:
Now we are only adding the properties that are not `lat/lon`, instead of removing them in a second step. Does this go into the direction you had in mind @ryanbaumann?

Reply:
beautiful @ustroetz.  I'm 👍  to merge when you're ready.